packages/tools/client-debugger/client-debugger-view/src/components/ContainerHistoryView.tsx

```diff
@@ -80,105 +88,69 @@ export function ContainerHistoryView(props: ContainerHistoryProps): React.ReactE
-			},
-		};
-
-		const iconStyle: IStackItemStyles = {
-			root: {
-				padding: 10,
-			},
-		};
-
-		historyViews.push(
-			<div key={`container-history-info-${i}`}>
-				<Stack horizontal={true} styles={accordionBackgroundColor}>
-					<StackItem styles={iconStyle}>
-						<Icon
-							iconName={
-								containerHistory[i].newState === ContainerStateChangeKind.Connected
-									? "PlugConnected"
-									: containerHistory[i].newState ===
-									  ContainerStateChangeKind.Attached
-									? "Attach"
-									: containerHistory[i].newState ===
-									  ContainerStateChangeKind.Disconnected
-									? "PlugDisconnected"
-									: containerHistory[i].newState ===
-									  ContainerStateChangeKind.Disposed
-									? "RemoveLink"
-									: containerHistory[i].newState ===
-									  ContainerStateChangeKind.Closed
-									? "SkypeCircleMinus"
-									: "Help"
-							}
-						/>
-					</StackItem>
-					<StackItem>
-						<div
-							key={`${containerHistory[i].newState}-${containerHistory[i].timestamp}`}
-						>
-							<b>State: </b>
-							{containerHistory[i].newState}
-							<br />
-							<b>Time: </b>
-							{wasChangeToday
-								? changeTimeStamp.toTimeString()
-								: changeTimeStamp.toDateString()}
-							<br />
-						</div>
-					</StackItem>
-				</Stack>
-			</div>,
-		);
-	}
+	// Columns for rendering audience history
+	const containerHistoryColumns = [
+		{ columnKey: "state", label: "State" },
+		{ columnKey: "time", label: "Time" },
+	];
+
+	const getBackgroundColorForState = (state: string): string => {
+		switch (state) {
+			case "connected":
+				return tokens.colorPaletteGreenBackground2; // green
+			case "disconnected":
+				return tokens.colorPaletteDarkOrangeBorderActive; // orange
+			case "closed":
+				return tokens.colorPaletteRedBorder1; // red
+			case "disposed":
+				return tokens.colorPaletteDarkRedBackground2; // dark red
+			case "attached":
+				return tokens.colorPaletteRoyalBlueBackground2; // blue
+			default:
+				console.log("Unknown state type for container!");
+				return tokens.colorBrandBackgroundPressed; // black
+		}
+	};
 
 	return (
-		<Stack
-			styles={{
-				root: {
-					height: "100%",
-				},
-			}}
-		>
-			<StackItem>
-				<h3>Container State History</h3>
-				<Stack
-					styles={{
-						root: {
-							overflowY: "auto",
-							height: "300px",
-						},
-					}}
-				>
-					<div style={{ overflowY: "scroll" }}>{historyViews}</div>
-				</Stack>
-			</StackItem>
-		</Stack>
+		<>
+			<Divider appearance="brand"> Audience State </Divider>
+			<Table size="small" aria-label="Audience history table">
+				<TableHeader>
+					<TableRow>
+						{containerHistoryColumns.map((column, columnIndex) => (
+							<TableHeaderCell key={columnIndex}>
+								{column.columnKey === "state" && <PlugConnected24Regular />}
+								{column.columnKey === "time" && <Clock20Regular />}
+								{column.label}
+							</TableHeaderCell>
+						))}
```

Comment:
Nit: I'm not sure we gain much by extracting the columns into an array where we then have to compare against each entry... what do you all think about just defining them here?

Reply:
https://react.fluentui.dev/?path=/docs/components-table--default

I believe, the code snippets provided in the official documentation prefers defining column key separately and followed this way as I thought this would keep the consistent code style for other developers working on UI task using `fluent-ui`. In addition, `AudienceView` incorporates a similar logic.